### DISCUSSION TITLE
以获取sql执行权限，执行插入语句报错

### DISCUSSION THREAD

**Discussion by Lovely-red-scarf** [hhyo/Archery#1673] [Q&A]
报错信息： Execute: Access denied; you need (at least one of) the SUPER, REPLICATION CLIENT privilege(s) for this operation (sql: SHOW MASTER STATUS;).
Execute: Can't found master binlog position.

**Comment by LeoQuote** [comment]
报错写的很清楚的, 要的权限都写了, 不明白的可以再看看项目的wiki
